src/main/java/core/basesyntax/strategy/impl/BalanceOperationImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy.impl;
```

Comment:
The package declaration should not be followed by an empty line. Please remove the empty line to adhere to Java style guidelines.